docs/post-processors/docker-import.mdx

```diff
@@ -22,6 +22,27 @@ registry.
 ## Basic Example
 
 <Tabs>
+<Tab heading="HCL2">
+
+```hcl
+source "docker" "example" {
+    image = "ubuntu:18.04"
+    export_path = "party_parrot.tar"
+}
+
+build {
+  sources = [
+    "source.docker.example"
+  ]
+
+  post-processor "docker-import" {
+    repository = "local/ubuntu"
+    tag = "latest"
```

Comment:
```suggestion
    tags = ["latest"]
```

Reply:
The `docker-import` post-processor only accepts a single tag, from the `tag` (`string`) field. The tag post-processor allows multiple with the `tags` (`list[string]`) field or the `tag` (`list[string]`) field (maintained for backward compatibility.